webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/domain/queries/CmdbQuery.scala

```diff
@@ -564,7 +564,7 @@ case object OrderedStringComparator extends TStringComparator {
 
 case object DateComparator extends LDAPCriterionType {
   override val comparators               = OrderedComparators.comparators.filterNot(c => c == Regex || c == NotRegex)
-  val fmt                                = "dd/MM/yyyy"
+  val fmt                                = "yyyy/MM/dd"
```

Comment:
I would not change that

Reply:
ignore my comment

Reply:
Why not use RFC3339 format ? If we are changing it at all? (ie `yyyy-MM-dd`)